llvm/test/tools/sycl-post-link/emit_imported_symbols.ll

```diff
@@ -0,0 +1,67 @@
+; This test checks that the -emit-imported-symbols option generates a list of imported symbols
+;
+; RUN: sycl-post-link -symbols -emit-imported-symbols -split=kernel -S < %s -o %t.table
+
+; RUN: FileCheck %s -input-file=%t_0.sym --check-prefixes CHECK-SYM-0
+; RUN: FileCheck %s -input-file=%t_1.sym --check-prefixes CHECK-SYM-1
+; RUN: FileCheck %s -input-file=%t_2.sym --check-prefixes CHECK-SYM-2
+
+;; Function names were chosen so that no function with a 'inside' in their function name is imported
```

Comment:
Nit: Please move the comment to the top; with the description. Thanks

Reply:
Comment moved.